aimmo-game/tests/functional/test_map_movements.py

```diff
@@ -0,0 +1,195 @@
+from unittest import TestCase
+
+from mock_world import MockWorld
+from simulation.location import Location
+from simulation import map_generator
+from tests.test_simulation.dummy_avatar import MoveEastDummy, MoveWestDummy, MoveNorthDummy, MoveSouthDummy, WaitDummy
+
+
+class TestMapMovements(TestCase):
```

Comment:
Are you testing the Map Movements, or the movements within the map? :thinking:

Reply:
:+1: